### PR TITLE
dairy_queen: modernise and rename to dairy_queen_us

### PR DESCRIPTION
**Pull request by davidhicks**

### CODE REVIEW

**Comment by iandees** [comment]
Can you spot a way to distinguish between the "Grill", "Chill", and "Grill+Chill" restaurants? Grill has burgers, etc., chill has just ice cream (likely seasonal hours), grill+chill has both.

**Comment by davidhicks** [comment]
> Can you spot a way to distinguish between the "Grill", "Chill", and "Grill+Chill" restaurants? Grill has burgers, etc., chill has just ice cream (likely seasonal hours), grill+chill has both.

Done :)

File: locations/spiders/dairy_queen_us.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                "Referer": "https://www.dairyqueen.com/",
+            },
+            data={"size": 10000, "query": {"bool": {"must": [{"term": {"contenttype": "locationDetail"}}]}}},
+        )
+
+    def parse(self, response):
+        for location in response.json()["contentlets"]:
+            item = DictParser.parse(location)
+            item["lat"], item["lon"] = location.get("latlong", ",").split(",", 2)
+            item.update(self.brands[location["conceptType"]])
```
Comment (Cj-Malone) [inline locations/spiders/dairy_queen_us.py:37]: Updating extras in this way wont work if we ever change DictParser to collect an extra, like fax address or twitter. Or like LinkedDataParser when started parsing hotel extras (stars).

Comment (davidhicks) [reply locations/spiders/dairy_queen_us.py:37]: Fixed in https://github.com/alltheplaces/alltheplaces/pull/6298 (I hope)

**Pull request merged**